apps/finance/app/src/components/Transfers.js

```diff
@@ -259,13 +196,12 @@ const Transfers = React.memo(({ tokens, transactions }) => {
             <TransfersFilters
               dateRangeFilter={selectedDateRange}
               onDateRangeChange={handleSelectedDateRangeChange}
-              tokenFilter={selectedToken}
               onTokenChange={handleTokenChange}
-              transferTypeFilter={selectedTransferType}
               onTransferTypeChange={handleTransferTypeChange}
-              compactMode={compactMode}
-              symbols={['All tokens', ...symbols]}
-              transferTypes={TRANSFER_TYPES_STRING}
+              tokenFilter={selectedToken}
+              transferTypeFilter={selectedTransferType}
+              transferTypes={READABLE_TRANSFER_TYPES}
```

Comment:
Perhaps this value should be controlled by `useFilteredTransfers()`, to encapsulate all the filtering logic to be within it.

It would be bad if we re-ordered the list but forgot to update it here or in that hook, which suggests we should encapsulate the concepts :).

Reply:
> Perhaps this value should be controlled by useFilteredTransfers(), to encapsulate all the filtering logic to be within it.

Now that I think about it, it makes sense, as `symbols` is handled by `useFilteredTransfers()` itself. Now, would we just import the labels and export them back with another name?

Reply:
Yep, that's what I would've done!